content/sql/concepts/commands/terms/contains/contains.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'CONTAINS'
+Description: 'Searches for matches to words or phrases.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+CONTAINS() is a predicate used to search for matches to words or phrases in a column.
+
+## Code Example
+
+`CONTAINS()` takes two arguements. A column, and the word/phrase to search for.
+
+```sql
+SELECT column_name
+FROM table_name
+WHERE CONTAINS(column_name, 'word');
+```
+
+```sql
+SELECT names
+FROM contacts
+WHERE CONTAINS(names, 'Paul');
```

Comment:
```suggestion
WHERE CONTAINS(first_name, 'Paul');
```

Reply:
I changed this to target the `first_name` column as it's more specific than `names` in my opinion. We can discuss if you want to try something else as well